src/aatoolbox/datasources/ecmwf_api/ecmwf_api_download_seasonal_monthly.py

```diff
@@ -42,14 +45,16 @@
 PRATE_DIR = "prate"
 
 
-def download_date(
+def download(
     # question: do we want to assume an iso3 here?
     # might want to e.g. download for a multi-country region or only a city
     iso3: str,
-    date_forec: Union[str, date],
     ecmwf_dir: Union[str, Path],
     iso3_gdf: gpd.GeoDataFrame,
+    date_list: List[Union[str, date]] = None,
```

Comment:
I feel like a list of dates is a fairly complex parameter for the user to provide. For the GloFAS API for instance, I just had parameters for the start and end dates, which can be set to the extent of the dataset

Reply:
Fair point. Changed it